week-3/E-array-map/exercise.js

```diff
@@ -4,11 +4,16 @@
 */
 
 // Complete this function -->
-function multiplyBy100(numbers) {}
+function multiplyBy100(numbers) {
+  return numbers * 100;
+}
 
 // DEBUG
 // Debug your function by
 // - calling it with some sample input
 // - logging the output to the console
 
+var numbs = [10, 20, 30];
+var numbsDoubled = numbs.map(multiplyBy100);
```

Comment:
`numbsDoubled` implies they are multiplied by 2 not 100 :slightly_smiling_face: .